dataprofiler/tests/profilers/profiler_options/test_base_inspector_options.py

```diff
@@ -1,10 +1,11 @@
 from dataprofiler.profilers.profiler_options import BaseInspectorOptions
-from dataprofiler.tests.profilers.profiler_options.test_boolean_option import \
-    TestBooleanOption
+from dataprofiler.tests.profilers.profiler_options.test_boolean_option import (
+    TestBooleanOption,
+)
```

Comment:
simliar comment around `isort` v `black` 🤔  @jakleh

Reply:
let's solve this post.